packages/angular-display/src/lib/display.module.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ModuleWithProviders,
+  NgModule,
+  Optional,
+  SkipSelf,
+} from '@angular/core';
+import { IsDisplayDirective } from './directive/is-display.directive';
+import { Breakpoints, breakpointsInjectionToken } from './model/breakpoints';
+
+@NgModule({
+  declarations: [IsDisplayDirective],
+  exports: [IsDisplayDirective],
+})
+export class DisplayModule {
+  public static forRoot(
+    breakpoints: Breakpoints
+  ): ModuleWithProviders<DisplayModule> {
+    return {
+      ngModule: DisplayModule,
+      providers: [
+        {
+          provide: breakpointsInjectionToken,
+          useValue: breakpoints,
+        },
+      ],
+    };
+  }
+
+  public constructor(@Optional() @SkipSelf() parentModule?: DisplayModule) {
```

Comment:
Don't think that this is correct, because you need to import the Module in other Sub-Modules of your app to be able to use the Directive.

Reply:
Fixed.